.github/workflows/rubocop.yml

```
@@ -14,9 +14,9 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Set up Ruby
-        uses: ruby/setup-ruby@e27aee156d42e38ff8a4c2fc97b125c4087cbcc7
+        uses: ruby/setup-ruby@b00551441a6fa3d528f3443dbff264a934b6e316
```

There was [another release](https://github.com/ruby/setup-ruby/releases/) in the meantime, which reminded me that I generally wanted to ask:

1. why the sha is pinned?
2. whether there are specific criteria, which version to pin?

I guess we could also do

```suggestion
        uses: ruby/setup-ruby@v1.85.0
```

or …

---

1. Sha is pinned because the "version" is based on a tag, which could be maliciously retagged in the future. Because actions effectively have write access to the codebase, if someone later got access to the setup-ruby repo and pushed and retagged their bad push as 1.85.0 then we'd be in trouble.
2. The latest sha is generally fine :)